### PR TITLE
[Bug][scaleph-ui-react] function hasPrivilege json parse error

### PR DESCRIPTION
**Pull request by hx23840**
if user first login,function hasPrivilege of auth.ts json parse error #219

## Brief change log
- fix  auth.ts function hasPrivilege json parse error

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If necessary, please update the documentation to describe the new feature.

### CODE REVIEW

**Review by kalencaya — DISMISSED** [review]

**Comment by kalencaya** [comment]
LGTM

**Comment by kalencaya** [comment]
Congratulation for your contributions and welcome to join scaleph!

Now scaleph is cultivating community ecosystem and trys to open team cooperation through zenhub, which is an excellent chrome plugin for github. Except that, scaleph will sponsor weekly online meeting on Tuesday 20:30.

Of course, you need not to trouble that must join some channel or pipeline, scaleph is open, equality and respect.

File: scaleph-ui-react/src/services/auth.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ export async function isUserLoggedIn() {
 }
 
 export function hasPrivilege(code: string) {
-  let pCodes: string[] = JSON.parse(localStorage.getItem(USER_AUTH.pCodes) || '');
+  let pCodes: string[] = JSON.parse(localStorage.getItem(USER_AUTH.pCodes) || 'null');
```
Comment (gleiyu) [inline scaleph-ui-react/src/services/auth.ts]: Please modify it to the following code
`  let pCodes: string[] = JSON.parse(localStorage.getItem(USER_AUTH.pCodes) + '');`

**Comment by gleiyu** [comment]
This is a data type conversion problem, empty string can't convert to json, but null value can works well.
`localStorage.getItem(USER_AUTH.pCodes) + ''` result is null value

**Comment by hx23840** [comment]
> This is a data type conversion problem, empty string can't convert to json, but null value can works well. `localStorage.getItem(USER_AUTH.pCodes) + ''` result is null value

Modified as required

**Review by kalencaya — APPROVED** [review]

**Pull request merged**